### ISSUE TITLE
MikTeX Console -- Admin mode maybe not correctly handled

### ISSUE THREAD

**Issue by Friedrich2** [MiKTeX/miktex#268]
- [x] I have read the [MiKTeX Contributing Guidelines](https://github.com/MiKTeX/miktex/blob/master/CONTRIBUTING.md)
My PC configuration:
Windows 10, most modern version of MikTeX, working in a non-admin account (thus, every program is started by default as non-admin)

Opening MikTeX Console in non-admin mode, I am informed unobtrusively that I have MikTeX installed for private use (Section: Operation Mode), but do not get the decision dialog offering the restart with elevated privileges.
This dialog in turn appears if I start the console in admin mode with admin rights already granted via the Windows UAC dialog.

Is this the correct behaviour? I had intuitively expected to

- get the restart offer in restricted mode, followed by the UAC prompt when choosing to restart and
- to be in admin mode every time I start the program as administrator automatically -- without additional restart offer.

Perhaps, this control flow is sensible working with an account with restricted rights to give an administrator the possibility to only change its private configuration. But in Windows 10, programs are always started with restricted rights normally, even in an admin accout, such that this problem should not exist.

In other words, I propose to antedate the restart offer into the main page appearing when started with restricted rights and to always start directly in admin mode when admin privileges were already granted.

**Comment by edocevoli** [comment]
Duplicate of #128
